source/Integrate/Mail_Clients/applemail.md

```diff
@@ -33,7 +33,7 @@ Then enter the server details:
 </p>
 ![Settings for Mail to use SendGrid SMTP]({{root_url}}/images/apple_mail_3.png "SendGrid SMTP server settings")
 
-When writing a message, there is a dropdown beneath the Subject field, where you can define the outgoing server you wish to use. This way, you can send individual messages to your regular SMTP, and the messages you need to send through SendGrid can go through us. Please note that when you select this, it will be remembered until you change the selection.
+When writing a message, there is a dropdown beneath the Subject field, where you can define the outgoing server you wish to use. By doing so, you can send individual messages to your regular SMTP and the messages you need to send through SendGrid can go through us. Please note that when you select the outgoing server, it will be remembered until you change the selection.
```

Comment:
To define the server you want to use, select the drop-down menu beneath the *Subject* field. This functionality allows you to send individual SMTP messages in addition to the messages you usually send through SendGrid. 

{% info %}
The server you select will remain the default outgoing server until you change it.
{% endinfo %}